downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md

```diff
@@ -273,24 +275,27 @@ configVar = {name = "Jack", city = "Colombo"}
 
 A new way has been introduced to support the binding of any resource function to a generic native method regardless of the resource path parameters. The generic native method should be defined with a `BArray` parameter, which represents all the path parameters. To avoid errors due to overloaded methods, it is recommended to define the parameter type constraints as well.
```

Comment:
```suggestion
A new way has been introduced to support the binding of any resource method to a generic native method regardless of the resource path parameters. The generic native method should be defined with a `BArray` parameter, which represents all the path parameters. To avoid errors due to overloaded methods, it is recommended to define the parameter type constraints as well.
```